src/discovery/kubernetes/Akka.Discovery.KubernetesApi/reference.conf

```diff
@@ -0,0 +1,41 @@
+######################################################
+# Akka Service Discovery Kubernetes API Config       #
+######################################################
+
+akka.discovery {
+  # Set the following in your application.conf if you want to use this discovery mechanism:
+  # method = kubernetes-api
+
+  kubernetes-api {
+    class = "Akka.Discovery.KubernetesApi.KubernetesApiServiceDiscovery, Akka.Discovery.KubernetesApi"
+
+    # API server, cert and token information. Currently these are present on K8s versions: 1.6, 1.7, 1.8, and perhaps more
```

Comment:
We're definitely going to need to do some documentation on this to explain how pods need to be configured in order to allow for them to query the API - it looks to me that by default it _should_ work but there will be situations where it doesn't.